test-end-to-end/android-maestro/common/member_login.yaml

```diff
@@ -0,0 +1,15 @@
+appId: com.toughra.ustadmobile
```

Comment:
I think there isn't really a different login flow for admin or anyone else. Let's have one flow for login that accepts a variable for the username and password. I think we should call it LOGIN_USER or USER etc - PERSONNAME is confusing (makes me think we are talking about the person's actual name, not the username).

Reply:
Got it Mike, I'll make changes ,Thank you